scripts/jenkins/qa_devstack.sh

```diff
@@ -157,4 +157,11 @@ tox -e smoke
 EOF
 fi
 
+if [ -z "${DO_UNSTACK}" ]; then
+    sudo -u stack -i <<EOF
+cd $DEVSTACK_DIR
+./unstack.sh
+EOF
+fi
```

Comment:
we might want to ignore cleanup errors here (the script is in -e mode so it would cause failures otherwise)